plugin/pom.xml

```diff
@@ -14,7 +14,7 @@
   <packaging>hpi</packaging>
 
   <name>Configuration as Code Plugin</name>
-  <description>Manage Jenkins master configuration as code</description>
+  <description>Manage Jenkins controller configuration as code</description>
```

Comment:
Maybe it's better to leave just "Jenkins" here. Agent configurations can be also managed on the controller side

Reply:
addressed in 46affecd3f251ffca636ab1d75bfa8d10cc3849b